package.json

```
@@ -101,6 +101,7 @@
     "react-apollo": "2.4.1",
     "react-autosuggest": "9.4.3",
     "react-beautiful-dnd": "10.0.4",
+    "react-csv": "^1.1.1",
```

Wir pinnen dependencies, i.e., brauchen immer die fixe Version ohne ^.

```suggestion
    "react-csv": "1.1.1",
```